cmd/lakectl/cmd/repo_list.go

```diff
@@ -19,11 +19,13 @@ var repoListCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		amount := Must(cmd.Flags().GetInt("amount"))
 		after := Must(cmd.Flags().GetString("after"))
+		prefix := Must(cmd.Flags().GetString("prefix"))
 		clt := getClient()
 
 		resp, err := clt.ListRepositoriesWithResponse(cmd.Context(), &apigen.ListRepositoriesParams{
 			After:  apiutil.Ptr(apigen.PaginationAfter(after)),
 			Amount: apiutil.Ptr(apigen.PaginationAmount(amount)),
+			Prefix: apiutil.Ptr(apigen.PaginationPrefix(prefix)),
```

Comment:
Again, params order - 
Better keep the same order (prefix, after, amount).

Same goes to other files.